lib/jetstream/api/object.ex

```diff
@@ -90,7 +95,7 @@ defmodule Jetstream.API.Object do
         nuid: nuid,
         size: size,
         chunks: chunks,
-        digest: "SHA-256=#{Base.encode64(digest)}"
+        digest: "SHA-256=#{Base.url_encode64(digest)}"
```

Comment:
The ADR-20 just specific `Base64` encoding, but I ran into some cases where the nats CLI was complaining about invalid base64 data. But when I checked in the go client it was actually doing URL-safe base64 encoding https://github.com/nats-io/nats.go/blob/97ccbd884fa75a3a5e86bef0ee4c9374dc0dd2b5/object.go#L317-L319